File: Examples/PHP/Images/GetImageWithFormat.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\PDF\PdfApi;
use Aspose\PDF\AsposeApp;

class Images {

	public $pdfApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->pdfApi = new PdfApi();
	}

	public function getImageWithFormat() {
        // Upload file to Aspose Cloud Storage
		$fileName = "SampleImage.pdf";
		Utils::uploadFile($fileName);

        $result = $this->pdfApi->GetImageWithFormat($fileName, $pageNumber=1, $imageNumber=1, $format="jpeg", $width = 100, $height = 100, $storage = "", $folder = "");
        $fh = fopen(realpath(__DIR__ . '/../../..') . '/Data/Output/Test.png', 'w');
        fwrite($fh, $result);
        fclose($fh);
	}
}

$images = new Images();
$images->getImageWithFormat();

?>
//ExEnd:

File: Examples/DotNET/CSharp/Images/GetImageWithFormat.cs
using System;
using Com.Aspose.PDF.Api;
using Com.Aspose.PDF.Model;
using Com.Aspose.Storage.Api;

namespace Images
{
    class GetImageWithFormat
    {
        public static void Run()
        {
            // ExStart:1
            PdfApi pdfApi = new PdfApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            string fileName = "Lenovo_Tablet_2_Datasheet.pdf";
            int? pageNumber = 1;
            int? imageNumber = 1;
            string format = "jpeg";
            int? width = 0;
            int? height = 0;
            string storage = null;
            string folder = null; 

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.PDF Cloud SDK API to get image with format
                ResponseMessage apiResponse = pdfApi.GetImageWithFormat(fileName, pageNumber, imageNumber, format, width, height, storage, folder);

                if (apiResponse != null && apiResponse.Status.Equals("Ok"))
                {                    
                    Console.WriteLine("Get Image with Format, Done!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}
